plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/WebPage.java

```diff
@@ -3,19 +3,12 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.pagefactory.FieldDecorator;
-import ru.sbtqa.tag.pagefactory.annotations.ActionTitle;
-import ru.sbtqa.tag.pagefactory.annotations.ActionTitles;
 import ru.sbtqa.tag.pagefactory.environment.Environment;
-import ru.sbtqa.tag.pagefactory.exceptions.WaitException;
-import ru.sbtqa.tag.pagefactory.web.actions.WebPageActions;
-import ru.sbtqa.tag.pagefactory.web.utils.WebExpectedConditionsUtils;
-import ru.sbtqa.tag.qautils.errors.AutotestError;
 
 /**
- * Contains basic ru.sbtqa.tag.pagefactory.mobile.actions in particular with web elements
- * If we want to extend this functional - inherit from this class
+ * Inherit your web page objects from this class
  */
-public abstract class WebPage extends DefaultPage {
+public abstract class WebPage implements Page {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Convert the abstract class "WebPage" into an interface. ([squid:S1610](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1610))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh46-jrFHbA8-Egpmz&open=AWZh46-jrFHbA8-Egpmz)

[ISSUE_KEY]: AWZh46-jrFHbA8-Egpmz